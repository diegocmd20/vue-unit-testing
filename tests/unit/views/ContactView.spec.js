import { shallowMount } from '@vue/test-utils';
import ContactView from '@/views/ContactView.vue';

describe('ContactView.vue', () => {
    it('renders the Contact page', () => {
        const wrapper = shallowMount(ContactView);
        expect(wrapper.text()).toContain('Contact Page');
    });

    it('matches the snapshot', () => {
        const wrapper = shallowMount(ContactView);
        expect(wrapper.html()).toMatchSnapshot();
    });
});
