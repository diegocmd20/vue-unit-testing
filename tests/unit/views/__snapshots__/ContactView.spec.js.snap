// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ContactView.vue matches the snapshot 1`] = `
<div>
  <h1>Contact Page</h1>
  <p>This is the Contact page.</p>
</div>
`;
